packages/migrations/src/migrations/5.43.0/001/ddb-es/index.ts

```diff
@@ -0,0 +1,279 @@
+                    values
+                }),
+                // Latest entry revision
+                this.ddbEntryEntity.putBatch({
+                    PK: partitionKey,
+                    SK: "L",
+                    id: this.createRevisionId(entryId),
+                    entryId,
+                    locale,
+                    location: {
+                        folderId: "root"
+                    },
+                    locked: false,
+                    meta: {},
+                    modelId: TASK_MODEL_ID,
+                    status: "draft",
+                    tenant,
+                    TYPE: "cms.entry.l",
+                    version: 1,
+                    webinyVersion: process.env.WEBINY_VERSION,
+                    modifiedOn: new Date().toISOString(),
+                    revisionCreatedOn: new Date().toISOString(),
+                    revisionModifiedOn: new Date().toISOString(),
+                    revisionSavedOn: new Date().toISOString(),
+                    savedOn: new Date().toISOString(),
+                    createdOn: new Date().toISOString(),
+                    values
+                })
+            ];
+
+            const rawDatas = {
+                modelId: TASK_MODEL_ID,
+                revisionCreatedOn: new Date().toISOString(),
+                revisionSavedOn: new Date().toString(),
+                revisionModifiedOn: new Date().toString(),
+                createdOn: new Date().toString(),
+                modifiedOn: new Date().toString(),
+                status: "draft",
+                meta: {},
+                entryId,
+                id: this.createRevisionId(entryId),
+                version: 1,
+                location: { folderId: "root" },
+                locale,
+                values: {
+                    "number@iterations": 0,
+                    "text@taskStatus": "pending",
+                    "text@definitionId": definitionId,
+                    "text@name": name
+                },
+                tenant,
+                locked: false,
+                webinyVersion: process.env.WEBINY_VERSION,
+                latest: true,
+                TYPE: "cms.entry.l",
+                __type: "cms.entry.l",
+                rawValues: {
+                    "object@input": input
+                }
+            };
+
+            const ddbEsItems = [
+                this.ddbEsEntryEntity.putBatch({
+                    PK: partitionKey,
+                    SK: "L",
+                    data: await getCompressedData(rawDatas),
+                    index: esGetIndexName({
+                        tenant,
+                        locale,
+                        type: TASK_MODEL_ID,
+                        isHeadlessCmsModel: true
+                    })
+                })
+            ];
+
+            await batchWriteAll({
+                table: this.ddbEntryEntity.table,
+                items: ddbItems
+            });
+
+            batchWriteAll({
```

Comment:
Missing await here.

Reply:
Not sure why my comment got lost but just wanted to say thanks for this!